src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/xlf/Resources.cs.xlf

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
```

Comment:
Since this is a non-shipping tool it might be better to disable localization rather than use resources to localize this.  Your call but I can't imagine a case for us to use this localization right now.

Reply:
Makes sense. Removed.